extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -601,7 +603,9 @@ export const AzureProvisionDialog: React.FC = () => {
 
   const getResources = async () => {
     try {
-      const resources = await getResourceList(currentProjectId, publishType);
+      const requireLUIS = requiredRecognizers.some((p) => p.requiresLUIS);
```

Comment:
why not move this into getResourceList instead of passing it as a parameter?

Reply:
Gotcha